scripts/database.js

```diff
@@ -277,7 +277,8 @@ export class Database {
         const store = this.#db
             .transaction('masternodes', 'readonly')
             .objectStore('masternodes');
-        return new Masternode(await store.get('masternode'));
+        const mnData = await store.get('masternode');
+        return mnData === undefined ? null : new Masternode(mnData);
```

Comment:
```suggestion
        return !mnData ? null : new Masternode(mnData);
```
Do not check explicitally for undefined, if for some reason `null` is stored, it may cause problems